sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_connections_operations.py

```diff
@@ -49,6 +50,7 @@ def __init__(
         self._credentials = credentials
         self._init_kwargs = kwargs
 
+    @distributed_trace
```

Comment:
We don't need the decorator here. Only on the public methods.

Reply:
removed